### ISSUE TITLE
Is there any reason not to publish this onto DefinitelyTyped?

### ISSUE THREAD

**Issue by ryutamaki** [ingenieux/aws-sdk-typescript#27]
`npm @types` with typescript 2.0 is really useful and I want to use this on that.

Do you think to add this into DefinitelyTyped??
If not, is there any specific reason for that?

**Comment by DmitryEfimenko** [comment]
Yeah, with introduction of `@types` having this unavailable in there is really inconvenient
+1

**Comment by sirwolfgang** [comment]
One possible blocker is that in good form an update to DT would require a bunch of tests to be brought over or created. Both this and DT seem to lack coverage.

**Comment by quantuminformation** [comment]
Would be nice if AWS could fund the maintenance of this.

**Comment by ehartford** [comment]
isnt it already there?  https://github.com/DefinitelyTyped/DefinitelyTyped/tree/types-2.0/aws-sdk

**Comment by alejodaraio** [comment]
You must use is https://www.npmjs.com/package/@types/aws-sdk

**Comment by aldrinleal** [comment]
If someome is able to submit a patch, it would be nice. Unfortunately, I need to maintain it in my spare time. And its not much time.

**Comment by aldrinleal** [comment]
@ehartford you're comparing a 100KB file with a 5.7MB file. So I guess its not the same :)

**Comment by jabrythehutt** [comment]
I'm not sure if there's a need to publish a version of this to DefinitelyTyped since the typescript definitions are already included in the latest version of the AWS SDK

**Comment by aldrinleal** [comment]
Yes, but last time I've checked - need to confirm, AWS had its own take on TypeScript. Need to confirm, but last time, they wrote this to me:

> Thank you for reaching out. As it turns out, we have actually been working on TypeScript support in the SDK (https://github.com/aws/aws-sdk-js/tree/support/typescript), and we’re getting pretty close to shipping an initial version of the feature. Our JavaScript engineer has looked at aws-sdk-typecript before and says it has components that add value not covered by our current implementation. We’d love to spend some time to dive deeper and tease out more details about how those can be integrated into the SDK.
 
So you want to give it a try

**Comment by ehartford** [comment]
this package is better than the one in DT now.  I agree this should be published to @types/aws-sdk

**Comment by aldrinleal** [comment]
Can you guys make a comparison against the one AWS started to supply a few weeks ago? I must admit this project is becoming an emotional burden and I don't want to fight an uphill battle by myself.

**Comment by jabrythehutt** [comment]
I've been using the latest versions of the AWS SDK (without @types/aws-sdk) for about a month now and have not come across anything that's missing - so don't see any reason for continuing with this project or using @types/aws-sdk. Anything that's missing should be added as an issue to the AWS SDK instead.
